src/compass/utils/age.py

```diff
@@ -573,6 +666,12 @@ def create_parser():
                           help='Padding margin around CR position detected in the geocoded SLC '
                                'image. Actual margin is 2*margin from left-to-right and from'
                                'top-to-bottom')
+    optional.add_argument('-r', '--azimuth-ramp', dest='az_ramp', default=True,
```

Comment:
```suggestion
    optional.add_argument('-r', '--azimuth-ramp', dest='apply_az_ramp', default=True,
```
nit - rename by adding a verb for clarity